chia/rpc/data_layer_rpc_client.py

```diff
@@ -15,8 +16,10 @@ async def get_value(self, store_id: bytes32, key: bytes) -> Dict[str, Any]:
         # TODO: better hinting for .fetch() (probably a TypedDict)
         return response  # type: ignore[no-any-return]
 
-    async def update_data_store(self, store_id: bytes32, changelist: Dict[str, str]) -> Dict[str, Any]:
-        response = await self.fetch("batch_update", {"id": store_id.hex(), "changelist": changelist})
+    async def update_data_store(
+        self, store_id: bytes32, changelist: Dict[str, str], fee: Optional[uint64]
```

Comment:
It's not required, but `Optional` often would get a default of `None`.  Should that particular be avoided here?

```suggestion
        self, store_id: bytes32, changelist: Dict[str, str], fee: Optional[uint64] = None
```